code/modules/projectiles/guns/projectile/shotgun.dm

```diff
@@ -78,7 +78,7 @@
 	..(user, allow_dump=1)
 
 //this is largely hacky and bad :(	-Pete
-/obj/item/gun/projectile/shotgun/doublebarrel/attackby(var/obj/item/A, mob/user)
+/obj/item/gun/projectile/shotgun/doublebarrel/attackby(var/obj/item/A, mob/living/user)
```

Comment:
Please avoid changing the types of args in proc headers, particularly in these highly overridden ones, it can make changes to mob vars unpredictable (status PR for example was a bitch due to people overriding the arg type). It would be preferrable to typecheck user and cast it to living if needed.